data/en/queryexecute.json

```diff
@@ -21,6 +21,7 @@
 		{"title":"Passing Query Parameters using Array", "description":"When passing with an array use the ? as a placeholder in your sql", "code":"qryResult = queryExecute(\"SELECT * FROM Employees WHERE empid = ? AND country = ?\", [1,\"USA\"]);", "result":""},
 		{"title":"Passing Query Parameters using Struct of Structs", "description":"If you need to pass the cfsqltype or other cfqueryparam attributes you can pass a struct for each param.", "code":"qryResult = queryExecute(\"SELECT * FROM Employees WHERE empid = :empid AND country = :country\", {country={value=\"USA\", cfsqltype=\"cf_sql_varchar\"}, empid={value=1, cfsqltype=\"cf_sql_integer\"}});", "result":""},
 		{"title":"Specifying the Datasource", "description":"If you have not defined the datasource using this.datasource in Application.cfc or want to explicitly specify the datasource use the third argument.", "code":"qryResult = queryExecute(\"SELECT * FROM Employees\", {}, {datasource=\"myDataSourceName\"});", "result":""},
+		{"title":"Combined Example", "description":"Complete example showing use of an sql statement, query parameters using Struct of Structs, while specifying the datasource .", "code":"sql = \"SELECT * FROM Employees WHERE empid = :empid AND country = :country\";\nqparams = structNew();\nqparams.empid = { value=1, cfsqltype=\"cf_sql_integer\" };\nqparams.country = { value=\"Canada\", cfsqltype=\"cf_sql_varchar\" };\noptions = { datasource=\"myDataSourceName\" };\nqryResult = queryExecute(sql, qparams, options);", "result":""},
```

Comment:
I thought about doing that. I even looked at some other examples in CFDocs. Most places are using explicit Struct/Array creation, and not implicit. So I went with explicit.

Reply:
As a rough guide, function calls are slower/need more performance than lexical tokens, so that's preferred for me. I don't know if @pfreitag looks at it the same way.

Reply:
Yeah, I agree with @shaedrich and personally prefer using `{}` over `structNew()` I suppose it made more sense to use `structNew()` several years ago where people might be been running on older versions of CF, but `{}` works for most people. Feel free to update this in another PR if you want.

Reply:
I never understood the usage of implicit/explicit in this context. Anyway, I too prefer the literal syntax, and would have further initialized it in one statement instead of using three.

Reply:
Created a new pull request with implicit notation. If implicit notation is the standard for CFDocs, that should be added to the CONTRIBUTING.md document.

Reply:
Absolutely. That's a good idea :+1: